cpp/include/cuml/ensemble/randomforest.hpp

```diff
@@ -157,7 +185,8 @@ void fit_treelite(const raft::handle_t& user_handle,
                   int n_unique_labels,
                   RF_params rf_params,
                   bool* bootstrap_masks,
-                  rapids_logger::level_enum verbosity);
+                  T* feature_importances,
+                  rapids_logger::level_enum verbosity = rapids_logger::level_enum::info);
```

Comment:
Why are we changing the default value for verbosity?

Reply:
Fixed!